Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/result/ExposureData.kt

```diff
@@ -0,0 +1,10 @@
+package de.rki.coronawarnapp.risk.result
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+
+data class ExposureData(
+    val date: Long,
```

Comment:
Shouldn't this be some kind of Date object rather than long?

Reply:
for calculation its better to use long here. But I adjusted the variable name to 'dateMillisSinceEpoch' to be more specific.
If the value is later needed for any output, it can be cast to any cultural or personal definitions of what time might be...

Reply:
What does the date represent, what is it used for?

Reply:
_ExposureData_ has been renamed to _AggregatedRiskPerDateResult_. It is a temporary object for further aggregations.
 _date_ value is now called _dateMillisSinceEpoch_ . it is needed e.g. for determining latest risk in history, for example:   _.maxOfOrNull { it.dateMillisSinceEpoch }_ (-> mostRecentDateForRisk function)
This could also be done private inner class, but due readability I put it here.